internal/quickstart/create_flag.go

```diff
@@ -29,7 +33,18 @@ func NewCreateFlagModel(client flags.Client, accessToken, baseUri string) tea.Mo
 		accessToken: accessToken,
 		baseUri:     baseUri,
 		client:      client,
-		textInput:   ti,
+		help:        help.New(),
+		helpKeys: keyMap{
+			Back: key.NewBinding(
+				key.WithKeys("esc"),
+				key.WithHelp("esc", "back"),
+			),
```

Comment:
we can't go back on the input step here though - with the new success page we will be able to but is there a way to hide it when we should the textinput View?

Reply:
I removed "back" from the help text for create flag.